src/api/db/index.js

```diff
@@ -105,6 +111,23 @@ export default function initBookshelf(config) {
         promises = [...promises, ...morePromises];
       }
 
+      let schools = this.schools();
+      schoolNames = await School.fetchAll();
+      schoolNames = schoolNames.serialize().map(row => row.name);
+
+      for (let name of labelNamesToAdd) {
+        if (schoolNames.indexOf(name) > -1) {
+          schoolNamesToKeep.push(name);
+        }
+      }
+
+      let school_tags = await Promise.all(schoolNamesToKeep.map(tag_name => School.createOrSelect(tag_name)));
+      let schoolPromises = school_tags.map(async (tag) => {
+        await schools.attach(tag)
+      });
+
+      promises = [...promises, ...schoolPromises];
+
```

Comment:
`schools` table has schools inserted manually via CSV

labels are always stored as labels

`posts_schools` is a relational table which relates posts labels to schools

In code, if labels which will be saved have the same name as an entry in `schoolNames`, then a relation of the post's label is generated in the `posts_schools` table.